packages/data-context/src/actions/ErrorActions.ts

```diff
@@ -22,7 +22,7 @@ export class ErrorActions {
       }
 
       if (d.baseError?.id === id) {
-        d.baseError === null
+        d.baseError = null
```

Comment:
This is why the base error wasn't getting cleared even after calling `resetErrorAndLoadConfig` 😅

Reply:
:facepalm: 

I wonder if there's an eslint rule for this. If not, writing one would be an interesting side project.